Added slash to BootstrapTest.php on line 16

goodnesskay opened the pull request:
I got an error while I was running a unit test on this program to get how the program works and the error pointed to the fact that a slash was missing on line 16 of BootstrapTest.php

goodnesskay committed:
Just made a slight change

goodnesskay committed:
Just made a change

goodnesskay committed:
new one

goodnesskay committed:
all

goodnesskay committed:
Corrections to error in terms of slashes

perfectmak commented:
First, please make your commit messages more descriptive. And secondly what error did you encounter before making a change to the unit test file?

Check the composer.json file and see the classmap for the test. You would see that it should work.

goodnesskay commented:
I got an error while I was running a unit test on this program to get how the program works and the error pointed to the fact that a slash was missing on line 16 of BootstrapTest.php

perfectmak commented:
What command did you use to run the unit tests?
And please can you send in a snippet of the error message?

goodnesskay committed:
Updated Readme.md file

I have updated the readme.md file. I worked on brief detail about the service,prerequisites,getting started, installation,built with,contributing ,license and authors. Please check the update and give feedback.
